README.md

```diff
@@ -70,6 +70,10 @@ If for some reason you need to deploy from your local machine, follow these inst
 
 ## Writing a blog
 
+> **NOTE:** Using generative AI in *assisting* writing is fine, but please don't use it to write entire posts. 
+> Used badly generative AI's has a tendency to use complex works and phrasing making
```

Comment:
```suggestion
> Used badly generative AI's has a tendency to use complex works and phrasing making
```

Used badly, generative AI has a tendency to use complex words and phrasing. This makes